.infra/application.properties

```diff
@@ -13,10 +13,13 @@ debezium.source.plugin.name=pgoutput
 debezium.source.heartbeat.interval.ms=60000
 debezium.source.topic.prefix=t
 debezium.source.tombstones.on.delete=false
-debezium.transforms=Reroute,ReadOperationFilter,PostsFilter
+debezium.transforms=Reroute,Notifications,ReadOperationFilter,PostsFilter
 debezium.transforms.Reroute.type=io.debezium.transforms.ByLogicalTableRouter
-debezium.transforms.Reroute.topic.regex=(.*)
+debezium.transforms.Reroute.topic.regex=^((?!\.notification_v2).)*$
```

Comment:
Additionally, would this match, our topic does not have `_v2` in its name?

Reply:
Or is this a table name match?

Reply:
This table name match. It took me a while to find the right regex but I tested everything locally. It was a tough fight. We no longer have v1 tables btw

Reply:
roger, roger! good then 👍